examples/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+java_library(
+    name = "example",
+    srcs = glob(["src/main/java/com/example/*.java"]),
+    deps = [
+      "@grpc_java//core",
+    ],
+    plugins = [
+      ":io_grpc_grpc_java_api_checker_plugin",
+    ],
+)
+
+java_plugin(
+    name = "io_grpc_grpc_java_api_checker_plugin",
+    deps = [
+        "@com_google_guava_guava//jar",
```

Comment:
I didn't think about guava version since this PR, but should we use `guava:19.0` ?

Reply:
Ah... This tool does not require the guava (It is an optional dependency), so the user could use any guava version.

Reply:
I’ve noticed that some dependencies in pom may not be needed, I’ll dig into it.